### PR TITLE
Feature-looks

### PR DESCRIPTION
**Pull request by Tchilo**
- Add background color to leaderboard 
- Add background color to "ul" list
- Add snapshot to README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Leaderboard
 
 > Building leaderboard project with webpack, Css, JavaScript. This project can be able to get and send data to an api. 
-
+![screenshot](./leaders.png)
 Built this porject using HTML ,CSS and Webpack .
```
Comment (EricMbouwe) [inline README.md:7]: - [x] README file should look professional and contains all that's needed for running the app with the API(s) it consumes. Please add a section for:
- API setup
- Instructions about where and how to generate any necessary keys, credentials, how to set, store, and generate them
- API endpoints consumed in this app

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
### Hi @Tchilo,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight!
- Good PR
- Good commits

# Required Changes ♻️
- [ ] There are some necessary informations that are present in your readme

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Tchilo) [reply README.md:7]: Thanks I have done it

**Review by EricMbouwe — APPROVED** [review]
### Hi @Tchilo, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**